ui/components/src/TimeSeriesTooltip/focused-series.ts

```diff
@@ -124,15 +146,30 @@ export function getFocusedSeriesData(
   const chartModel = chart['_model'];
   const yAxisInterval = chartModel.getComponent('yAxis').axis.scale._interval;
 
-  // tooltip trigger area gets smaller with more series, increase yAxisInterval multiplier to expand nearby series range
   const seriesNum = chartData.timeSeries.length;
-  const yBuffer = seriesNum > TOOLTIP_MAX_ITEMS ? yAxisInterval * 0.5 : yAxisInterval * 5;
+
+  // tooltip trigger area gets smaller with more series, increase yAxisInterval multiplier to expand nearby series range
+  const yBuffer =
+    seriesNum > SHOW_MORE_NEARBY_SERIES_LIMIT
```

Comment:
i might call this `SHOW_FEWER_SERIES_LIMIT` because as the number of series grows from 1 --> more, when it reaches a certain point, we show fewer series. 

again up to you / not a problem as is!